components/raftstore/src/store/fsm/store.rs

```diff
@@ -292,16 +292,21 @@ where
 {
     fn notify(&self, apply_res: Vec<ApplyRes<EK::Snapshot>>) {
         for r in apply_res {
-            self.router.try_send(
-                r.region_id,
+            let region_id = r.region_id;
+            if let Err(e) = self.router.force_send(
+                region_id,
                 PeerMsg::ApplyRes {
                     res: ApplyTaskRes::Apply(r),
                 },
-            );
+            ) {
+                error!("failed to send apply result"; "region_id" => region_id, "err" => ?e);
```

Comment:
Seems the error log is printed only when the channel is closed. 
So is it necessary to add this error log?

Reply:
A little is better than none. 🤣